packages/datetime/src/dateRangeInput.tsx

```diff
@@ -62,6 +63,12 @@ export interface IDateRangeInputProps extends IDatePickerBaseProps, IProps {
     contiguousCalendarMonths?: boolean;
 
     /**
+     * Props to pass to ReactDayPicker.
+     * See API documentation [here](http://react-day-picker.js.org).
```

Comment:
Let's include the full prop description wherever this prop appears. Note that we may have a slightly different blacklist for `DateRangeInput` and `DateRangePicker `than for `DateInput` and `DatePicker`.

Reply:
yep. `dateInput` and `dateRangeInput` are not done yet though